Dockerfile.main

```diff
@@ -1,5 +1,6 @@
 FROM docker.io/library/eclipse-temurin:23-jdk-ubi9-minimal AS builder
 RUN microdnf -y install git-core maven rpm-libs
+RUN mkdir /opt/rpm-libs && cp /usr/lib64/librpm.so.9 $(ldd /usr/lib64/librpm.so.9 | awk '{print$3}') /opt/rpm-libs
```

Comment:
What do you think about using `/usr/lib64/librpm.so.*` instead so we don't have to worry about soname bumps?
It should work the same way.

Reply:
:+1: That should work.

Reply:
```dockerfile
RUN mkdir /opt/rpm-libs && cp /usr/lib64/librpm.so.* $(ldd /usr/lib64/librpm.so.* | awk '{print$3}' | sort -u) /opt/rpm-libs
```
avoids `cp: warning: source file 'XXX' specified more than once`

Reply:
Also, would you please put that line on the bottom since it is the most likely to be changed?

Reply:
It will change only when `microdnf -y install ... rpm-libs` changes.
This is there reason it's right after after the `microdnf` call.